apps/designer/tsconfig.json

```
@@ -1,6 +1,6 @@
 {
   // Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
-  "extends": "../../node_modules/@webstudio-is/tsconfig/remix.json",
+  "extends": "../../packages/tsconfig/remix.json",
```

maybe we can actually use the proper package name now? 
```suggestion
  "extends": "@sebstudio-is/tsconfig/remix.json",
```

---

Nope, remix still can't resolve it